vite.config.js

```diff
@@ -10,7 +10,7 @@ export default defineConfig(({ command, mode }) => {
 			__MAPBOX_TOKEN__: JSON.stringify(env.MAPBOX_TOKEN),
 		},
 		plugins: [preact()],
-
+		base: '/gtfs-rt-inspector',
```

Comment:
What does this line do? Why the leasing `/`? Why does it not just default to the current directory?

Reply:
As it loads assets, they are loaded from root, thus on GH pages from /gtfs-rt-inspector

see https://vitejs.dev/guide/build#public-base-path 

you may customize it using params for example ... 

you prefer i remove it? i found it easier, it should change much locally

Reply:
I don't understand how it works locally when running `vite`? Will it this option not apply then? Or will it serve assets under `/gtfs-rt-inspector`?

Reply:
when you go to http://localhost:5173/ you are then redirected to http://localhost:5173/gtfs-rt-inspector?feedSyncInterval=30&feedSyncStopped=false&feedUrl&focusedTripId&focusedVehicleId&shapeIdsByTripIdUrl=%2Fshape-ids-by-trip-id.json&shapeUrl=%2Fshapes%2F%7Bshape_id%7D.geo.json&view=log

assets are served from http://localhost:5173/gtfs-rt-inspector/index.js for example.

I find it's better, because it mimics the way it will work once deployed...

The other option is to remove this from config, add it a build time in the ci with `--base=/gtfs-rt-inspector`

tell me what you prefer and i'll update it

Reply:
The way you did it now sounds good.